lib/miq_automation_engine/service_models/mixins/miq_ae_saved_url_mixin.rb

```diff
@@ -0,0 +1,7 @@
+module MiqAeSavedUrlMixin
+  extend ActiveSupport::Concern
+  def saved_url=(url)
```

Comment:
@martinpovolny If we are specifically using this for the custom button URL it would be better if that name reflected that.  The method name exposed here does not have to match the backend model method, but that usually makes things easier so we should seriously consider naming at each level.